libraries/Microsoft.Bot.Builder.Core.Extensions/RecognizerResult.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Core.Extensions
+{
+    /// <summary>
+    /// Recognizer return value.
+    /// </summary>
+    public class RecognizerResult
```

Comment:
I added JsonProperties to that class with camelCasing that seem to have been lost.

Reply:
Fixed